About.html

```diff
@@ -0,0 +1,107 @@
+                <li class="menu-choice blog"><a href="" class="link-desk">Blog</a></li>
+                <li class="menu-choice news"><a href="" class="link-desk">News</a></li>
+                <li class="menu-choice sponsor"><a href="" class="link-desk">Sponsor</a></li>
+                <li class="menu-choice call" id="contact"><a href="" class="link-desk link-desko">Call Us</a></li>
+            </ul>
+        </nav>
+        <ul class="options">
+            <li class="choice"><a href="index.html" class="go-to-page">Home</a></li>
+            <li class="choice "><a href="About.html" class="go-to-page">About</a></li>
+            <li class="choice"><a href="#" class="go-to-page">Blog</a></li>
+            <li class="choice"><a href="#" class="go-to-page">Sponsor</a></li>
+            <li class="choice"><a href="#" class="go-to-page">News</a></li>
+            <li class="choice"><a href="#" class="go-to-page">Contact</a></li>
+        </ul>
+        <section class="summary">
+            <p class="greeting">Hello! Future Musicians,</p>
+            <div class="a-title"><h1 class="summary-title">DARKO MUSIC COURSE 2022 SUMMER INTAKE</h1></div>
+            <div class="in-short">
+                <p class="text">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Possimus explicabo nemo laboriosam minima alias, reiciendis a! Vel iste cupiditate provident, inventore exercitationem, recusandae quidem dicta dolores soluta aut aspernatur ipsum!</p>
+            </div>
+            <div class="email-us">
+                <p class="text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro veritatis sapiente quam, alias quod pariatur.</p>
+                <p class="our-email">darkomusic@rda.org</p>
+            </div>
+        </section>
+        <section class="hist-sect">
+            <div class="hist-title">
+                <h2 class="real-title">Darko Music 2022 History</h2>
+            </div>
+            <div class="underline"></div>
+            <div class="hist-summary">
+                <p class="text">Darko Music was founded by Christian I.B in 2020. It started as record studio and later expanded to online class and music promoting company.</p>
+            </div>
+            <div class="big"><img src="IMAGES/darko-logo.png" alt="logo" class="logo-big"></div>
+        </section>
+        <section class="past">
+            <div class="past-title">
+                <h2 class="real-title">See Past Darko Promotions</h2>
+            </div>
+            <div class="underline"></div>
+            <div class="sum">
+                <p class="text">
+                    Take a look at the past two promotions since the creation of Darko Music as online class.
+                </p>
+            </div>
+            <div class="past-images">
+               <div class="past past1">
+                   <div class="color-overlay"></div>
+                   <p class="year">Darko Music 2020</p>
+               </div>
```

Comment:
- [ ] Kindly raise the size of the image and please move them more to the center following [design guideline](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Keep coding 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/163674671-3dfc6970-22db-4168-8588-c2895a3fa42b.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/163674680-3c7cde5f-0e5e-440c-b5c2-1726e9d9af64.png)